aws/resource_aws_appmesh_virtual_node.go

```diff
@@ -155,6 +189,13 @@ func resourceAwsAppmeshVirtualNode() *schema.Resource {
 										Elem: &schema.Resource{
 											Schema: map[string]*schema.Schema{
 												"service_name": {
+													Type:     schema.TypeString,
```

Comment:
Not going through `Deprecated` cycle here, straight to `Removed` as AppMesh is a preview service.